packages/scripts/scripts/test-playwright.js

```diff
@@ -28,9 +28,29 @@ const {
 } = require( '../utils' );
 
 /**
- * Internal dependencies
+ * @typedef {import('../../env/lib/config/load-config').WPConfig} WPConfig
+ */
+
+/**
+ * Loads any configuration from a given directory.
+ *
+ * @type {(configDirectoryPath: string) => Promise<WPConfig> | null}
  */
-const loadConfig = require( '../../env/lib/config/load-config' );
+let loadConfig = null;
+
+try {
+	// First, try to load the package installed from among the optional peerDependencies.
+	loadConfig = require( '@wordpress/env/lib/config/load-config' );
+} catch ( error ) {
+	try {
+		// If the above fails, then we're likely in the Gutenberg monorepo and can reference the file directly.
+		loadConfig = require( '../../env/lib/config/load-config' );
```

Comment:
Is this code path actually triggered in this repo? `@wordpress/env/lib/config/load-config` won't exist?

Reply:
Humm. I guess it does in the monorepo. No error is thrown when running `npm run test:e2e` in the `try` block.

Reply:
Well, that is, the code in the `try` block works in the monorepo, and presumably it would work if `@wordpress/env` were installed as a peer dependency. So then this nested `try` isn't needed apparently.

Reply:
Removed in 79c602eed0b80119093505a38cd6b6fa0c908208